frontend/tests/utils/helpers.ts

```diff
@@ -1,14 +1,30 @@
-import { TEST_URL } from "tests/common/constants";
+import { TEST_EMAIL, TEST_URL } from "../common/constants";
 import { getText } from "./selectors";
 
 export async function goToPage(url: string = TEST_URL) {
   await page.goto(url);
 }
 
-export async function login(url: string = TEST_URL) {
-  await goToPage(url);
```

Comment:
What's the reason to remove goToPage() here? Thank you!

Reply:
I forget the exact reasoning, but I believe I did it because the login action is available at any page and I don't believe login should own any page navigation as it's unrelated to the action itself and would just be a side effect

Reply:
ahh I see! 

Right now I think logging in will always dump the user back to the homepage regardless of which page the user is previously on before clicking the login button, because FE is not setting `redirect` param for the `/login` API call based on the user's current url. So `login()` needs to be called before we navigate to the destination page

For example, the following code will end up being on the homepage, even if the intention is to stay on the collection page after logging in:

```ts
await goToPage('/collections/foo');
await login();

// the user will still be on homepage at this point
```

Unless we update FE's call to `/login` to include the user's current URL as `redirect`

Otherwise we'd need to do:

```ts
await goToPage();
await login();
await goToPage('/collections/foo');
```

or:

```ts
await goToPage('/collections/foo');
await login();
await goToPage('/collections/foo');
```

which is more redundant than just keeping `await goToPage();` in `login()`, so we only need to:

```ts
await login();
await goToPage('/collections/foo');
```

?

Reply:
Somewhere along the line I thought we were redirecting, I guess I made that assumption at some point because we only have a single non-authenticated page. I'll implement:
```
await login();
await goToPage('/collections/foo');
```
in the meantime